### PR TITLE
Create kizuki

### PR DESCRIPTION
**Pull request by GwenZappy**

### CODE REVIEW

File: kizuki.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- **Variety of Flavors**: Whether you're a fan of bold, savory tonkotsu broth or prefer a lighter shoyu or shio base, Kizuki Ramen offers a wide variety of flavors to suit every palate. The menu includes options for both meat lovers and vegetarians, ensuring there's something for everyone.
+
+- **Fresh and High-Quality Ingredients**: The ingredients used at Kizuki Ramen are of top-notch quality, ensuring a delicious and satisfying meal. The meats are tender, the vegetables are fresh, and the broth is carefully prepared to perfection.
+
+- **Customizable Options**: If you like to tailor your ramen to your taste, Kizuki Ramen allows you to customize your bowl by choosing the type of noodles, spice level, and additional toppings. This flexibility ensures that you get a bowl of ramen that's just the way you like it.
+
+- **Friendly and Knowledgeable Staff**: The staff at Kizuki Ramen are friendly, attentive, and always willing to help with recommendations or any inquiries you may have. They enhance the overall dining experience with their excellent service.
+
+Service options: Dine-in · Takeout · No-contact delivery
```
Comment (ianchen06) [inline kizuki.md]: Can you add a heading called “Store information” and make the following into a markdown list 

```
- hours: 9am-5pm
- Address: 320 pine st
- Menu: kizuki.com
```

**Review by ianchen06 — DISMISSED** [review]
Looks delicious I would love to add this to the database. Can you please address the comments about adding a heading for store information?

**Comment by ianchen06** [comment]
Hi Gwen,
The file location is incorrect. Seems like you are storing it in the root directory

**Comment by GwenZappy** [comment]
I see! I will change it

**Pull request closed without merging**